clients/web/src/containers/syndicated-article/syndicated-article.js

```diff
@@ -88,36 +82,18 @@ export function SyndicatedArticle({ queryParams = validParams, locale }) {
   // modify rendered elements when query params are passed in
   const { mobile_web_view, premium_user } = queryParams
   const isMobileWebView = mobile_web_view === 'true'
-  const isPremiumUser = premium_user === 'true' || isPremium === '1'
+  const isPremiumUser = premium_user === 'true' || (isPremium === '1' && 1 === 2)
```

Comment:
Is this hardcoded to false-y just for testing?

Reply:
Yes... There are some better ways to do this that are a bit more sustainable.  We should probably remove this so it doesn't leak